elasticsearch-cluster-CCR/test/README.md

```diff
@@ -0,0 +1,57 @@
+# Run CCR (Cross Cluster Replication) test on Elasticsearch
```

Comment:
please explain either in here, or in comments in each file, what each file does, along with their configurations